README.md

```diff
@@ -69,3 +69,15 @@ $ forge --help
 $ anvil --help
 $ cast --help
 ```
+
+## Validate Upgrade Safety 
+1. Checkout to the deployed implementation commit 
+2. Copy the YieldDistributor to test/upgrades/<version>/YieldDistributor.sol
+3. Checkout to upgrade candidate version (A version that is strictly higher than the version in the previous step)
+4. Update the version in the options object of the script/upgrades/ValidateUpgrade.s.sol script
+5. Run `forge clean && forge build && forge script script/upgrades/ValidateUpgrade.s.sol`
+6. If script is run succesfully , proceed , otherwise address errors produced by the script unti no errors are produced.
+
+## Test Upgrade with Calldata Locally 
+1. Amend the `data` variable `script/upgrades/UpgradeYieldDistributor.s.sol` to match desired data 
```

Comment:
Amend the `data` variable in `script/upgrades/UpgradeYieldDistributor.s.sol` to match desired data

Reply:
What's the diff here? 🤔

Reply:
addressed with 6a374af